code/datums/mood_events/generic_negative_events.dm

```diff
@@ -302,3 +302,23 @@
 	description = span_boldwarning("It isn't ending... it isn't ending, come on...\n")
 	mood_change = -18
 	timeout = 3 MINUTES
+
+/datum/mood_event/bad_touch_bear_hug
+	description = "I just got squeezed way too hard."
+	mood_change = -3
+	timeout = 2 MINUTES
+
+/datum/mood_event/rippedtail
+	description = "<span class='warning'>I ripped their tail right off, what have I done!</span>\n"
```

Comment:
does `span_warning()` work in strings...? i don't know

Reply:
if something expects its assignment to be static (i.e. defined as part of the object) macros don't work

Reply:
span_warning() should work here.

It's an issue if a macro has a non-constant in it, then it would fail. `#define span_warning(str) ("<span class='warning'>" + str + "</span>")` is the define, which is fine

but if it were defined as `#define span_warning(str) ("<span class='warning'>[str]</span>")` then it would fail because [str] is an interpolated operator.